### PR TITLE
Created profile pages for pod mates

### LINKED ISSUE #9: Profile page for each fellow
When you click on each fellows card, it sends you to a separate page which includes more detail such as:

- Description
- Favourite programming languages
- Interests outside of technology
- University

### PR DESCRIPTION
**Pull request by akanshaaa19**
## Please, go through these steps before you submit a PR.

- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

Fixes #9 

We have created a profile page for each pod member, it shows their picture, university, favorite programming languages and
interests outside of technology.

Screenshots:
image

image

image

Co-authored by @suppixie

### CODE REVIEW

File: .bashrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+bundle install --path vendor/bundle
```
Comment (akshgpt7) [inline .bashrc]: You don't need to commit the `.bashrc` file, can you please omit it from the PR.

**Review by akshgpt7 — CHANGES_REQUESTED** [review]
This looks awesome! Thanks a lot fo working on this!
Can you just address my review comment and resolve the merge conflict?
It'll be good to merge after that.

**Comment by akanshaaa19** [comment]
Hi i have made some new changes, check it out again. I think it is ready to be merged

File: .bashrc (hunk 1 above)
Comment (suppixie) [reply .bashrc]: Okay, we have resolved it. The bash file has been removed.

**Comment by ttn-ttn** [comment]
@akshgpt7 the profile pages aren't showing up neither in the preview for this PR nor in the deployed website, maybe this should be reverted?

**Comment by akanshaaa19** [comment]
Hii @ttn-ttn it is showing, check once again
Here is a screenshot 
IMG_4325

**Comment by ttn-ttn** [comment]
Okay, sorry! I must be blocking something, then.

**Pull request merged**